### ISSUE TITLE
language dropdown automatically gets selected when it receives focus: A11y_Aspire Dashboard_Langauge_OnFocus.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/aspire#10128]
### GitHubTags
#A11ySev2; #A11yTCS;#DesktopWeb; #Win32;#.NET Project system for CSharp and VB;#BM_AspireDashboard_Web_June25;#Win32;#WCAG3.2.1;#OnFocus;#A11yMAS;

### Environment Details:
Application Name: Aspire Dashboard
Edition Windows 11 Enterprise
Version 24H2
OS build 26100.4351

### Repro Steps:

1. Open Visual studio.
2. Open a aspire project and run it.
3. A new window will open in the chromium edge, TAB to "Settings" tab item and hit ENTER key.
4. Verify that language dropdown automatically gets selected when it receives focus

### Actual Result:
language dropdown automatically gets selected when it receives focus

### Expected Result:
On receiving the focus language options shouldn't get automatically selected and user can have to press enter/space to choose the language.

### User Impact:
Keyboard users won't be able to select the required language, if on receiving the focus if it gets selected.

**Comment by adamint** [comment]
Interestingly, I can no longer reproduce this. @AnKushSingh05 please verify with the latest version of the aspire code.

**Comment by adamint** [comment]
Unable to reproduce on latest code in main.

**Comment by adamint** [comment]
I did however find a related issue that is fixed in the above PR.
